### ISSUE TITLE
Bug: User status is not in sync for Channels versus Boards

### ISSUE THREAD

**Issue by wiggin77** [mattermost/focalboard#4273]
## Steps to reproduce the behavior

1. Open Mattermost in a browser (Chrome or FF)
2. Select Boards icon on RHS
3. Create boars if needed, or select existing board
4. Observe board opened in new browser tab
5. Observe the user status is offline in Boards tab, and online in Channels tab

## Expected behavior

User status icon should accurately reflect user status for all products.

## Screenshots (optional)
![presence-status-bug](https://user-images.githubusercontent.com/5024605/205140684-7563348c-7963-4532-b3a5-460418f31571.gif)

## Edition and Platform

 - Edition: Mattermost Boards - bug bash server
 - Version: 7.6
 - FF & Linux

## Additional context (optional)

Add any other context about the problem here, and any notes about the severity:
* Sev 1: Affects critical functionality without a workaround
* Sev 2: Affects major functionality with a difficult or non-obvious workaround
* Sev 3: Affects minor, non-critical functionality

**Comment by wuwinson** [comment]
@harshilsharma63 Do you recall why this happens?

**Comment by harshilsharma63** [comment]
@wuwinson yup, the logic to load user's status in the header bar is not triggered when in products. I have a old PR - https://github.com/mattermost/mattermost-webapp/pull/9288 but the fix wasn't good enough to merge.

Its a product issue, not a boards issue.

**Comment by wuwinson** [comment]
Thanks @harshilsharma63! I'll open a JIRA ticket for this instead.
